<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorClass</class>
 <widget class="QMainWindow" name="CalculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>697</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>496</width>
    <height>697</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>496</width>
    <height>697</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#282828</string>
  </property>
  <property name="iconSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; border:none</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_results">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;color:white; text-align: right;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFac05; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_all">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>172</green>
          <blue>5</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFac05; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFac05; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>540</y>
      <width>201</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: white</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFac05; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFac05; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ctg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3b3b3b; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>ctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3b3b3b; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3b3b3b; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3b3b3b; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3b3b3b; color: white; border:none;</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="actionDeveloper_Sultanov_Alexander_Version_1_0"/>
     <addaction name="actionVersion_1_0"/>
    </widget>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
  </widget>
  <action name="actionDeveloper_Sultanov_Alexander_Version_1_0">
   <property name="text">
    <string>Developer: Sultanov Alexander.</string>
   </property>
  </action>
  <action name="actionVersion_1_0">
   <property name="text">
    <string>Version: 1.0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
